tutorials/btp-integration-suite-execute-batch-payload/btp-integration-suite-execute-batch-payload.md

```diff
@@ -0,0 +1,377 @@
+
+    <!-- border -->![Send query from Postman](Blank-Query-Send.png)
+
+3. Before proceeding to the next step, remove Content Modifier from your integration flow. We had added it to create a database successfully.
+
+
+
+> For simplicity, we will use Postman to push the payload, later we will also see how to use Groovy Script to push the payload.
+
+
+### Execute Batch Payload
+
+1. In your integration flow, edit the settings for JDBC Receiver adapter. Under **Connections** tab, select the **Batch Payload** checkbox and **Batch Operation** as **Atomic**. We are doing this because Batch supports XML payload in INSERT, UPDATE and DELETE modes only. Native SQL queries are supported with prepared statements only.
+
+    <!-- border -->![Batch Payload Settings](Batch-Payload-Setting.png)
+
+2. Open your HTTP client (in this case Postman) to create a payload. Enter the **Username** and **Password** under **Authorisation** tab.
+
+    <!-- border -->![Postman- Authorization](Postman-Authorization.png)
+
+3. To create Payload, we will use INSERT mode and create a payload (with table name `samplejdbctest` ) to test Batch Payload scenario. To use these modes, you must follow the syntax. **Send** the following payload from **Request > Body** tab of postman:
+
+    ```
+    <root>
+    	<insert_statement1>
+    		<dbTableName action="INSERT">
+    			<table> samplejdbctest</table>
+    			<access>
+    				<emp_id>1</emp_id>
+    				<emp_name hasQuot="Yes">v XXXXX</emp_name>
+
+    			</access>
+    		</dbTableName>
+    	</insert_statement1>
+    	<insert_statement2>
+    		<dbTableName action="INSERT">
+    			<table>samplejdbctest</table>
+    			<access>
+    				<emp_id>2</emp_id>
+    				<emp_name hasQuot="Yes"> g28</emp_name>
+
+    			</access>
+    		</dbTableName>
+    	</insert_statement2>
+    </root>
+    ```
+
+      <!-- border -->![Sample Playload](Payload.png)
+
+Notice that the JDBC adapter provides the content from the database in a specific format as described under [ Defining XML Documents for Message Protocol XML SQL Format](https://help.sap.com/docs/SAP_NETWEAVER_750/5cf7d2de571a45cc81f91261668b7361/2e96fd3f2d14e869e10000000a155106.html?version=7.5.4&locale=en-US)
+
+To check the Message Processing Log (MPL) of this integration flow from your tenant, go to **Monitoring** > **Overview** > **All Integration flows**. Choose the latest record with your artefact name.
+
+  <!-- border -->![Payload Processed](Payload-Processed.png)
+
+> Ensure to check the time stamp to determine which record to check.
+
+
+
+### Atomic Batch Operation
+
+Atomic Batch Operation doesn't allow an action to be performed on the payload if an error is encountered. To verify that, follow these steps:
+
+
+1. Copy the above payload and modify just one record and make it unique. For example, here we are making the **`emp_id`** (of second record) as 3. Thus, making that record unique, but the first record is duplicate. Send this query.
```

Comment:
Don't say "we" when addressing the author. Say "you" instead!

Reply:
done